exercise_012_clustered_sudoku_solver/src/main/scala/akkapi/cluster/sudoku/SudokuProblemSender.scala

```diff
@@ -34,6 +34,8 @@ class SudokuProblemSender private (sudokuSolver: ActorRef[SudokuSolver.Command],
   private val solutionWrapper: ActorRef[SudokuSolver.Response] =
     context.messageAdapter(response => SolutionWrapper(response))
 
+  context.log.info("Dit is een demo ~~>")
```

Comment:
Should this be in english?

Reply:
I removed that line of code (shouldn't have been there anymore when I submitted the PR).